### ISSUE TITLE
YOLOV5 precision 

### ISSUE THREAD

**Issue by hiiksu** [ultralytics/yolov5#3394]
## ❔Question My precision is going CRAZY!!!
Hi, I'm using YOLOV5s to run machine learning. Datasets are using train-2600 valid-710 tests-65 tests. Precision is ridiculously low. Datasets are using yolo-mark and classes are two. I think my dataset is problematic.
I did it while listening to the lecture. There's no problem with the lecturer's dataset, but my dataset seems to have a problem. I tried to do epochs from 500 to 3,000 times. The results are similar. Is there a solution?
python train.py --img640 --batch16 --epochs500 data custum.yaml --weights yolov5s.pt
It's my code 
The result is approximately 0.0002 per cent 
Please help me 
## Additional context
I'm use Windows and Anaconda prompt

**Comment by hiiksu** [comment]
![results](https://user-images.githubusercontent.com/85096196/120157218-be7b4a80-c22d-11eb-8fa3-dda2876fe7d6.png)
This is my results

**Comment by glenn-jocher** [comment]
@hiiksu your labels are incorrect. You have no losses displayed which means you have no labels to learn from.

**Comment by hiiksu** [comment]
@glenn-jocher First of all, thank you for replying. But I think it's well done, can you check it for me?
![image](https://user-images.githubusercontent.com/85096196/120265936-ea550980-c2db-11eb-85a1-d6abc063d237.png)
![image](https://user-images.githubusercontent.com/85096196/120265988-fb9e1600-c2db-11eb-9c95-cdd1a7797028.png)

**Comment by glenn-jocher** [comment]
@hiiksu follow Train Custom Data tutorial below to get started, first start with COCO128 and then transition to your dataset.

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [TorchScript, ONNX, CoreML Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)

**Comment by hiiksu** [comment]
@glenn-jocher Do I have to rebuild the dataset?

**Comment by glenn-jocher** [comment]
@hiiksu in general your dataset looks a bit strange as there is no mosaic and you seem to have used some 3rd party augmentation method which I would not recommend.

For best results, just train normally with all default settings using default unmodified code.

**Comment by hiiksu** [comment]
@glenn-jocher  Thank you. Then I'll do it again with the basic setting. Thank you again.
